cid/main.py

```diff
@@ -66,26 +73,3 @@ def latest_azure_image(
 @app.get("/versions")
 def versions(db: Session = Depends(get_db)) -> list:  # noqa: B008
     return crud.find_available_versions(db)
-
-
-@repeat(every(24).hours)
-def self_update_image_data() -> None:
```

Comment:
Would a regular update of image data still make sense? 
This could be configurable in some way so that air gapped instances wouldn't try to reach our servers.

Or should we move to a "pull on demand"?
E.g. downloading data, mounting it into the container and restart / trigger a reload function.

Reply:
I know we've talked about having daily updated builds available with fresh data, but I could see a scenario in which it would be easier for customers to just inject new data instead of pulling a new container and moving it into an air gapped box. WDYT?